src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -1,23 +1,20 @@
 package net.masterthought.cucumber;
 
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.io.Reader;
+import java.io.*;
```

Comment:
I prefer not to use wildcard, if you don't mind

Reply:
No problem i'll change this back